react/features/app/middleware.js

```diff
@@ -0,0 +1,55 @@
+import {
+    CONNECTION_ESTABLISHED,
+    getURLWithoutParams
+} from '../base/connection';
+import { MiddlewareRegistry } from '../base/redux';
+
+MiddlewareRegistry.register(store => next => action => {
+    switch (action.type) {
+    case CONNECTION_ESTABLISHED:
+        return _connectionEstablished(store, next, action);
+    }
+
+    return next(action);
+});
+
+/**
+ * Notifies the feature app that the action {@link CONNECTION_ESTABLISHED} is
```

Comment:
That's the middleware of the feature app so the function notifies the feature app that the specific action is being dispatched. We have such a comment on a lot of middleware so, if it's wording that bothers you, let me know how you want it and I'll try to change it in every middleware.